xp/identityserver/Dockerfile

```diff
@@ -51,17 +51,21 @@ RUN Install-SitecoreConfiguration -Path $Env:SIF_CONFIG `
     -Package c:/Files/$Env:IDENTITYSERVER_PACKAGE `
     -SqlDbPrefix $Env:SQL_DB_PREFIX `
     -SitecoreIdentityCert "Identity"`
-    -LicenseFile "c:/Files/license.xml" `
+    -LicenseFile "c:/license/license.xml" `
     -SiteName $Env:SITE_NAME `
     -SqlCoreUser $Env:SQL_USER `
     -SqlCorePassword $Env:SQL_SA_PASSWORD `
     -SqlServer $Env:SQL_SERVER `
     -PasswordRecoveryUrl http://$Env:SITE_NAME `
     -AllowedCorsOrigins http://$Env:SITECORE_SITE_NAME `
-    -ClientSecret "SIF-Default"
+    -ClientSecret "SIF-Default" `
+    -Skip "StartAppPool", "StartWebsite" 
```

Comment:
Why do we need to skip those two tasks?

Reply:
It will not start with a dummy license, and start is not necessary during build.